typings.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "*.json" {
+    const value: any;
+    export default value;
+}
```

Comment:
No

Reply:
Why @szenzaro ? It looks like a very clean and extremely efficient solution in pure angular style. Avoid myriad http calls...